### ISSUE TITLE
colorpicker throwing onClick

### ISSUE THREAD

**Issue by VeraZab** [josdejong/jsoneditor#569]
Hi, 
I was trying the colorpicker option, looks awesome in the example.
![screen shot 2018-08-23 at 10 18 54 am](https://user-images.githubusercontent.com/4257572/44531115-f688f280-a6bd-11e8-996a-84fb2b32fceb.png)

So from my understanding, it's supposed to be available by default as the `colorPicker` option is true by default, correct.

I have the following jsoneditor options on my editor, and it does not show up.
```
 const options = {
            onChange: this.handleEditorChange,
            navigationBar: false,
            onValidate: this.validateJson,
            modes: ['tree', 'code'],
            onEditable: this.onEditable,
            history: false,
            sortObjectKeys: false,
            statusBar: false,
            onModeChange: this.onModeChange,
            onError: this.onError,
            colorPicker: true,
        };

this.jsonEditor = new JSONEditor(this.refs.jsoneditor, options);
```
![screen shot 2018-08-23 at 10 15 38 am](https://user-images.githubusercontent.com/4257572/44531195-2a641800-a6be-11e8-9c60-5ddc672529a2.png)

My guess would be that maybe it's because I'm using custom validation?
I'm not sure.

Let me know if I can provide more details.
Thanks!

**Comment by josdejong** [comment]
Thanks for reporting @VeraZab .  The color picker should indeed work by default, without any configuration needed.

Just to rule out that this issue is related to your data structure itself: if you open your JSON document in the web site https://jsoneditoronline.org, does it show color pickers there or not?

To check if it is related to custom validation, you could turn off custom validation and see if the issue still occurs. I tried in `examples/18_custom_validation.html` and there the color picker works fine.

To figure out whether it's a bug in the JSONEditor itself or something or related to the customizations you made in the JSONEditor we should see if we can reproduce this issue in a small JSBin or JSFiddle. Would it be easy for you to take one of the provided examples of the JSONEditor and insert your configuration and data there, see if that gives the same issue?

**Comment by VeraZab** [comment]
Closing, I believe its our old css files that are causing errors.
Will reopen if that doesn't end up being the problem.
Thanks for the debugging suggestions!

**Comment by josdejong** [comment]
:+1: good to hear it's working properly now

**Comment by josdejong** [comment]
This should be fixed in `v5.24.3`. If not please let met know.

https://github.com/josdejong/jsoneditor/blob/master/HISTORY.md#2018-08-29-version-5243
